install.sh

```diff
@@ -230,6 +296,10 @@ main() {
         echo "with a symlink from ${LATEST}/opengrep"
     fi
 
+    # Always update the symlink to point to the requested version
+    rm -f "${LATEST}" || exit 1
+    ln -s "${INST}" "${LATEST}" || exit 1
+
```

Comment:
It seems like this was "borrowed" into https://github.com/opengrep/opengrep/commit/8935a5951851e6d28639a2f1e807350453dfa051
```suggestion
```